financialaid/tasks.py

```diff
@@ -0,0 +1,30 @@
+"""
+Periodic task that updates currency exchange rates.
+"""
+import requests
+from django.conf import settings
+
+from financialaid.models import CurrencyExchangeRate
+from micromasters.celery import async
+
+
+@async.task
+def update_currency_exchange_rates():
+    """
+    Updates all CurrencyExchangeRate objects to reflect latest exchange rates from
+    Open Exchange Rates API (https://openexchangerates.org/).
+    """
+    url = "{url}latest.json?app_id={app_id}".format(
+        url=settings.OPEN_EXCHANGE_RATES_URL,
+        app_id=settings.OPEN_EXCHANGE_RATES_APP_ID
+    )
+    resp = requests.get(url).json()
+    latest_rates = resp["rates"]
+    for currency_exchange_rate in CurrencyExchangeRate.objects.all():
```

Comment:
Can you move this currency update code to a separate function in `api.py`? And we should also decorate that function with `@transaction.atomic` to make sure all exchange rates update at the same time